src/Microsoft.AspNet.Diagnostics/project.json

```diff
@@ -2,16 +2,16 @@
     "version": "1.0.0-*",
     "description": "ASP.NET 5 Middleware for error handling, error pages, and diagnostics information.",
     "dependencies": {
+        "Microsoft.AspNet.Diagnostics.Interfaces": "1.0.0-*",
```

Comment:
I came home early today because I wasn't feeling well, so feel free to be unconvinced if you think I'm mistaken :smile: (And then I'll just blame my not feeling well!)